app/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,38 @@
+'use client'
+
+import React from 'react';
```

Comment:
You could just import `useState` if that's the only thing you need. Something like: 

`import { useState } from 'react'`